### PR TITLE
Chapter 6

### PR DESCRIPTION
**Pull request by nuriman1**

### CODE REVIEW

**Comment by nuriman1** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class User < ApplicationRecord
+  validates :name, presence: true, length: { maximum: 50 }
```
Comment (manhnt-0960) [inline app/models/user.rb]: move constant like 50, 255 to settings.yml file

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  get 'users/new'
```
Comment (manhnt-0960) [inline config/routes.rb]: remove this line

File: app/models/user.rb (hunk 1 above)
Comment (nuriman1) [reply app/models/user.rb]: done

File: config/routes.rb (hunk 2 above)
Comment (nuriman1) [reply config/routes.rb]: done

**Comment by nuriman1** [comment]
ready

**Comment by manhnt-0960** [comment]
merged

**Pull request merged**